tests/SecurityUpdatesTest.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Unish;
+
+/**
+ * Tests "pm:security" commands for D8+.
+ * @group commands
+ * @group slow
+ * @group pm
+ */
+class SecurityUpdatesTest extends CommandUnishTestCase {
+
+  /**
+   * Test that insecure packages are correctly identified.
+   */
+  function testInsecurePackage() {
+    $this->drush('pm:security', array(), array(), NULL, NULL, self::EXIT_ERROR);
+    $this->assertContains('One or more of your dependencies has an outstanding security update. Please apply update(s) immediately.', $this->getErrorOutput());
+    $this->assertContains('Try running: composer require drupal/alinks:^1.1 --update-with-dependencies', $this->getErrorOutput());
+    $this->assertContains('drupal/alinks', $this->getOutput());
+    $this->assertContains('1.0.0', $this->getOutput());
+    $this->assertContains('1.1', $this->getOutput());
```

Comment:
Use --format=json option and then your asserts can be more precise. Example: https://github.com/drush-ops/drush/blob/747ae29a782f8c51ab3eb3d19268811eed6fb7ff/tests/configTest.php#L49-L51